yt_dlp/extractor/_extractors.py

```diff
@@ -787,6 +787,7 @@
     IHeartRadioIE,
     IHeartRadioPodcastIE,
 )
+from .ilpost import IlPostExtractorIE
```

Comment:
`Extractor` is redundant
```suggestion
from .ilpost import IlPostIE
```